sentences/de/climate_HassClimateGetTemperature.yaml

```diff
@@ -7,4 +7,4 @@ intents:
           - "Wie hoch ist [<area>] die Temperatur"
           - "Wie hoch ist die Temperatur (von | vom) <name>"
           - "Wie hoch ist die Temperatur vom Thermostat <area>"
-          - "Wie warm ist es <area>"
\ No newline at end of file
+          - "Wie (warm | heiß | kalt) ist es <area>"
```

Comment:
if you just want to ask "Wie warm ist es?", than the `area` needs to be optional

```suggestion
          - "Wie (warm | heiß | kalt) ist es [<area>]"
```

Reply:
Resolved with #445